File: jmccc/src/main/java/org/to2mbn/jmccc/launch/LaunchException.java
package org.to2mbn.jmccc.launch;

/**
 * Thrown when failing to launch the game.
 *
 * @author yushijinhun
 */
public class LaunchException extends Exception {

    private static final long serialVersionUID = 1L;

    public LaunchException() {
    }

    public LaunchException(String message, Throwable cause) {
        super(message, cause);
    }

    public LaunchException(String message) {
        super(message);
    }

    public LaunchException(Throwable cause) {
        super(cause);
    }

}

File: jmccc/src/main/kotlin/org/to2mbn/jmccc/launch/LaunchException.kt
package org.to2mbn.jmccc.launch

open class LaunchException : Exception {
	constructor() : super()
	constructor(message: String?) : super(message)
	constructor(cause: Throwable?) : super(cause)
	constructor(message: String?, cause: Throwable?) : super(message, cause)
}
